lib/auth/native/native.go

```diff
@@ -289,6 +289,15 @@ func (k *Keygen) GenerateUserCertWithoutValidation(c services.UserCertParams) ([
 		cert.Permissions.Extensions[teleport.CertExtensionDisallowReissue] = ""
 	}
 
+	for _, extension := range c.CertificateExtensions {
+		// TODO(lxea): update behavior when non ssh, non extensions are supported.
```

Comment:
Do we have any plans to work on that?

Reply:
I dont know if there are concrete plans for this, it wasnt needed for the github key extensions though.